From 0d33ba469f976782a1586f2b3b1eebe26f2f8486 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:08 -0400
Subject: [fs] btrfs: Fix typos in printk

Message-id: <1407179109-13714-2-git-send-email-sandeen@redhat.com>
Patchwork-id: 86959
O-Subject: [RHEL7.1 PATCH 001/422] btrfs: Fix typos in printk
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Masanari Iida <standby24x7@gmail.com>

commit 77d84ff87e9d38072abcca665ca22cb1da41cb86
Author: Masanari Iida <standby24x7@gmail.com>
Date:   Mon Dec 9 00:22:53 2013 +0900

    treewide: Fix typos in printk

    Correct spelling typo in various part of kernel

    Signed-off-by: Masanari Iida <standby24x7@gmail.com>
    Acked-by: Randy Dunlap <rdunlap@infradead.org>
    Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/tests/free-space-tests.c b/fs/btrfs/tests/free-space-tests.c
index 6fc8201..790f118 100644
--- a/fs/btrfs/tests/free-space-tests.c
+++ b/fs/btrfs/tests/free-space-tests.c
@@ -101,7 +101,7 @@ static int test_extents(struct btrfs_block_group_cache *cache)
 
  ret = btrfs_remove_free_space(cache, 2 * 1024 * 1024, 4096);
  if (ret) {
-  test_msg("Error removing middle peice %d\n", ret);
+  test_msg("Error removing middle piece %d\n", ret);
   return ret;
  }
 
-- 
1.7.1